lib/dao.js

```diff
@@ -433,6 +439,7 @@ DataAccessObject.create = function(data, options, cb) {
           if (err) return cb(err);
 
           if (connector.create.length === 4) {
+            options = addTransaction(Model, options);
             connector.create(modelName, obj.constructor._forDB(context.data), options, createCallback);
           } else {
             connector.create(modelName, obj.constructor._forDB(context.data), createCallback);
```

Comment:
`Model.getDataSource().isTransaction / transaction` will only be set if transactions are supported and `dataSource.transaction(execute)` was used. So I don't think there is a need for exceptions?